src/olympia/migrations/871-untranslate-canned-responses.sql

```diff
@@ -0,0 +1,25 @@
+/* Add the new columns and drop the constraints so we can clear the translations.
+   Local DB does have foreign key constraints, but AMO prod doesn't, so commenting
+   out those lines.*/
```

Comment:
I have those locally:

```
  CONSTRAINT `name_refs_id_8865ac11` FOREIGN KEY (`name`) REFERENCES `translations` (`id`),
  CONSTRAINT `response_refs_id_8865ac11` FOREIGN KEY (`response`) REFERENCES `translations` (`id`)
```

Pretty weird that they don't exist on dev/stage/prod.